### PR TITLE
Fix classpath variable modifier.

### PR DESCRIPTION
**Pull request by azenla**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -56,14 +56,14 @@ tasks.withType(FindBugs) {
 applicationDistribution.from new File(project.projectDir, "/dslink.json")
 
 startScripts {
-    def configureClasspathVar = { findClasspath, pathSeparator, variable, line ->
+    def configureClasspathVar = { findClasspath, pathSeparator, slash, variable, line ->
         line = line.replaceAll(~/^${findClasspath}=.*$/) { original ->
-            original += "${pathSeparator}${variable}lib"
+            original += "${pathSeparator}${variable}lib${slash}*"
```
Comment (azenla) [inline build.gradle:61]: Fixes incorrect positioning of the star expansion.

**Comment by Pacane** [comment]
Does this work on all platforms now?

**Comment by azenla** [comment]
@Pacane Yes, it correctly generates the right paths for Windows, Mac, and Linux.

**Review by a-hansen — COMMENTED** [review]
All I did was check the generated scripts pre and post and the change looks good to me.  I didn't run it though, but it seems obvious enough.  Can't comment on the Groovy, that's all magic to me.

Can you get online with Julian, Joel or I and show runs both before and after.  That's how I handle tests/reviews with Julian and it seems effective and efficient.

**Review by Pacane — APPROVED** [review]

**Pull request merged**